### PR TITLE
add resource config for fastqc process

### PR DESCRIPTION
**Pull request by phue**
mini-PR to add a resource configuration for `FastQC`
closes https://github.com/nf-core/methylseq/issues/143

### CODE REVIEW

**Review by ggabernet — APPROVED** [review]
Hi @phue,
this is fine like this but we try to harmonize the process requirements labels like [here](https://github.com/nf-core/tools/blob/master/nf_core/pipeline-template/conf/base.config) using Labels so that one does not need to specify the requirements for each process in the `base.config`. So whenever possible, just adding the `label process_low` for example inside that process then defines the requirements that correspond to the `withLabel:process_low` definition in `base.config`.

**Comment by phue** [comment]
Thanks! Yup we'll switch to process labels with dsl2 :+1:

**Review by ewels — APPROVED** [review]

**Comment by ewels** [comment]
Tests should pass once the template sync PR is merged I guess..

File: conf/base.config [hunk 1]
```diff
@@ -26,6 +26,11 @@ process {
     memory = { check_max( 64.GB * task.attempt, 'memory') }
     time = { check_max( 36.h * task.attempt, 'time') }
   }
+  withName:fastqc {
```
Comment (ewels) [inline conf/base.config:29]: Trying to trigger the tests again.
```suggestion
  withName:fastqc { 
```

File: conf/base.config [hunk 2]
```diff
@@ -26,6 +26,11 @@ process {
...
```
Comment (ewels) [inline conf/base.config:29]:
```suggestion
  withName:fastqc {
```

**Pull request merged**